### ISSUE TITLE
Bulgarian Translation

### ISSUE THREAD

**Issue by PrOxYsTaR** [steverichey/DMLocalizer#2]
```
Игра на
Задвижвано от
натисни ляво или дясно
отпусни ляво или дясно за да започнете
отпусни ляво или дясно за да изтриете
напълнете лентата за да изтриете данните
изтриване на запаметени данни
опити
дистанция
секунди
медали
нива
монети
трофеи
връх
връхове
Край
отблокиран трофей
пауза
време
Задържи ляво или дясно за рестарт
Не мърдай
ниво нагоре
край на играта
```

P.S there are two words zenith and zeniths that right now i don't know their translation to bulgarian but i will look into it and will update with more information :)

**Comment by steverichey** [comment]
Thanks! Some other terms you could consider would be peak, acme, pinnacle, or summit. Hope that helps!

**Comment by PrOxYsTaR** [comment]
You want me to translate "peak", "acme", "pinnacle" and summit or what?

**Comment by steverichey** [comment]
No, I'm just offering those as alternatives with similar meanings to "zenith".

**Comment by PrOxYsTaR** [comment]
Ah okay :+1:

**Comment by PrOxYsTaR** [comment]
I have updated my comment with the complete localization you asked :)

**Comment by steverichey** [comment]
@PrOxYsTaR: Do these look good for the title images? I just updated the localizer to support Bulgarian characters, sorry they didn't work before.

![dont_bg](https://cloud.githubusercontent.com/assets/3226635/3921374/f3dbb826-23c0-11e4-8a4c-f9c627debd8d.png)
![levelup_bg](https://cloud.githubusercontent.com/assets/3226635/3921373/f3d80226-23c0-11e4-8b44-e9ee6f7a5a52.png)
![zenith_bg](https://cloud.githubusercontent.com/assets/3226635/3921372/f3d6951c-23c0-11e4-9a32-cdee10695403.png)
![gameover_bg](https://cloud.githubusercontent.com/assets/3226635/3921375/f3e05cc8-23c0-11e4-96ec-304ab15cff7b.png)

**Comment by PrOxYsTaR** [comment]
Yeah they look very good i like them :+1:

**Comment by steverichey** [comment]
Hey @PrOxYsTaR, how would you like to be credited?

**Comment by steverichey** [comment]
Implemented in `9ec4bd89`
